tests/resources/functions/timeout/index.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
-return function ($request, $response) {
-    sleep(5);
+return function ($context) {
+    sleep(60);
```

Comment:
Let's reduce this as much as possible

Reply:
I think high number is good, to prevent false failing tests. We are not testing accuracy here, just end-to-end flow to ensure we still can get timeout error.
More accurate timeout check is done in open runtimes tests.

Reply:
We can optimize tests later as part of the work to split tests per service.